src/controller/CHIPDevice.cpp

```diff
@@ -475,35 +492,118 @@ bool Device::GetAddress(Inet::IPAddress & addr, uint16_t & port) const
     return true;
 }
 
-CHIP_ERROR Device::EstablishCASESession()
+void Device::OperationalCertProvisioned()
```

Comment:
nit: Suggest `OnOperationalCertProvisioned` as I think this reads a bit easier and we use that convention elsewhere.